habito/habito.py

```diff
@@ -140,6 +140,41 @@ def add(name, quantum, units):
                       created_date=datetime.now(), quantum=quantum,
                       units=units, magica="")
 
+@cli.command()
+@click.argument("id", type=click.INT)
+@click.option('--name', '-n', help="The new name (leave empty to leave unchanged)")
+@click.option('--quantum', '-q', help="The new quantum (leave empty to leave unchanged)", type=click.FLOAT)
+def edit(id, name, quantum):
```

Comment:
I like the idea of edit. Is it possible to prevent misuse (e.g. user edits quantum only for today due to an impulse)? Similar to discussion in #7, should we allow scheduling of `quantum` edits? Name edits are fine to be done right away.